torch/utils/benchmark/examples/callgrind.py

```diff
@@ -0,0 +1,36 @@
+"""Demonstrate Callgrind collection in both C++ and Python."""
+
+import torch
+import torch.utils.benchmark as benchmark_utils
+
+
+def main():
+    python_timer = benchmark_utils.Timer(
+        stmt="y = x + torch.ones((1,))",
+        setup="x = torch.ones((1,))",
+    )
+
+    cpp_timer = benchmark_utils.Timer(
+        stmt="auto y = torch::add(x, torch::ones({1}));",
+        setup="torch::Tensor x = torch::ones({1});",
+        language="C++",
+    )
+
+    python_times = python_timer.blocked_autorange(min_run_time=5)
```

Comment:
This is not really part of this PR, but I really can't tell what `blocked_autorange` means just looking at the method name.

Reply:
Are you familiar with the `timeit.Timer.autorange` method?